scripts/windows/setup/Install-Prerequisites.ps1

```diff
@@ -45,33 +40,10 @@ if (-not $BuildRepositoryLocalPath) {
     $BuildRepositoryLocalPath = DefaultBuildRepositoryLocalPath
 }
 
-$All = -not $Dotnet -and -not $Python -and -not $Nuget
+$All = -not $Python -and -not $Nuget
```

Comment:
reminder: See similar comment from linux script equivelent. Dotnet is getting installed externally so I removed the install logic and flags from this script. Do a sanity check to make sure these prereqs are truly needed.